CHANGES.md

```diff
@@ -90,6 +90,15 @@ breaking changes, and mappings for the large list of deprecated functions.
 
 [Migration guide]: https://github.com/openssl/openssl/tree/master/doc/man7/migration_guide.pod
 
+### Changes between 3.0.1 and 3.0.2 [?? ??? 2022]
```

Comment:
Date template: xx XXX xxxx

Our release scripts depend on it (which could be changed, if course)